images/pom.xml

```diff
@@ -76,17 +89,27 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.wildfly.plugins</groupId>
-                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <groupId>${server.maven.plugin.groupId}</groupId>
+                        <artifactId>${server.maven.plugin.artifactId}</artifactId>
+                        <version>${server.maven.plugin.version}</version>
                         <configuration>
                             <!-- some tests check for the provisioned galleon layers -->
                             <record-provisioning-state>true</record-provisioning-state>
+                            <channels>
```

Comment:
For EAP/XP to work properly channel needs to be used. For wildfly it does not need as sources are synced. But I do not know how to configure channel based on what product is run. So lets use channel also for WF. Channel should be generated during WF build so hopefully that will be transparent for WF.

Reply:
Two options:

1) Set the 'wildfly.channels' maven property; if set wildfly-maven-plugin will use it. I believe eap-maven-plugin does not support this though, so that may mean this is not an option.

2)  The eap profile can amend the plugin config.

Reply:
I will investigate your proposal, but to make clear Is this blocker for merging. As CI checks are green current configuration works properly.

Reply:
@mchoma No it's not a blocker, but if anyone relies on setting these properties externally in order to control the manifests when testing WildFly, they should assume that will break.

Reply:
https://github.com/mchoma/wildfly-cloud-tests/pull/1 moves this out of the default profile, into the eap profile

Reply:
I have merged kabirs proposal which implemented option number 2